eradiate/thermoprops/tests/test_afgl1986.py

```diff
@@ -22,7 +22,7 @@
         "us_standard",
     ],
 )
-def test_make_profile(model_id: str) -> None:
+def test_make_profile(mode_mono, model_id: str) -> None:
```

Comment:
Please don't add type hints to test functions.

Reply:
I'll remove them!